common/changes/@microsoft/api-documenter/patch-1_2023-09-20-22-22.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
```

Comment:
Let me know if there is a particular format you prefer for the change files. Happy to update as needed.